android/quest/src/main/java/org/smartregister/fhircore/quest/data/report/measure/MeasureReportRepository.kt

```diff
@@ -83,6 +89,29 @@ constructor(
     val measureReport = mutableListOf<MeasureReport>()
     try {
       withContext(dispatcherProvider.io()) {
+        val measure =
+          fhirEngine
+            .search<Measure> { filter(Measure.URL, { value = measureUrl }) }
+            .first()
+            .resource
+
+        // TODO move to Sync
```

Comment:
is this todo still relevant? I guess we want to come back to this in the future, but we're ok for now, we can create an issue to reference here if there is something we want to come back to to change

Reply:
Created the tracker issue and added the link to the comments: https://github.com/opensrp/fhircore/issues/3019